iot_packet_verifier/src/verifier.rs

```diff
@@ -110,11 +108,14 @@ where
                     })
                     .await
                     .map_err(VerificationError::InvalidPacketWriterError)?;
-                self.config_server
-                    .disable_org(report.oui)
-                    .await
-                    .map_err(VerificationError::ConfigError)?;
             }
+            match remaining_balance {
+                Some(remaining_balance) if remaining_balance >= minimum_allowed_balance => {
+                    self.config_server.enable_org(report.oui).await
```

Comment:
does it matter if the org is already enabled and we keep calling enable_org each time there is a balance change ?  Is there any performance or other consideration with this ?

Reply:
The `config_server` has a cache of enabled_clients and only calls `iot_config` if it needs to.

Reply:
iot_config also treats this idempotently so it's safe to call directly against the service as well